modules/apps/object/object-web/src/main/java/com/liferay/object/web/internal/object/entries/display/context/ObjectEntryDisplayContextImpl.java

```diff
@@ -1215,6 +1256,21 @@ private DTOConverterContext _getDTOConverterContext() {
 			_themeDisplay.getUser());
 	}
 
+	private Object _getFormattedDate(
+			String fieldName, ObjectEntry objectEntry, Object rawDate)
+		throws PortalException {
+
+		return _getValue(
```

Comment:
Is there a better way to have it formatted? Instead converting it to DDMFormField?

Reply:
You can isolate the _getValue method to have independent of ddmFormField